javagen/pom.xml

```diff
@@ -94,32 +104,6 @@
               </execution>
             </executions>
           </plugin>
-
-          <plugin>
-            <artifactId>maven-resources-plugin</artifactId>
-            <version>3.2.0</version>
-            <executions>
-              <execution>
-                <id>copy-resources</id>
-                <phase>generate-sources</phase>
-                <goals>
-                  <goal>copy-resources</goal>
-                </goals>
-                <configuration>
-                  <outputDirectory>${basedir}/src/main/resources</outputDirectory>
-                  <resources>
-                    <resource>
-                      <directory>${basedir}/../preprocessor</directory>
-                      <includes>
-                        <include>data-plane.md</include>
-                      </includes>
-                      <filtering>false</filtering>
-                    </resource>
-                  </resources>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
```

Comment:
We still need this. It syncs the data-plane.md in resource to the module root.

The data-plane.md in resources is used in runtime to compare with autorest options (in runtime), to figure out which option is changed, so that to add it to swagger/autorest.md output file during codegen.

I should have better doc on this block, though :-)

I think in current flow, it could just copy the file to `${basedir}/../javagen`

Reply:
Does this need to be added back? Or do I need to just move the copied `data-plane.md` files into resources for `javagen` and `androidgen`?

Reply:
Ah, I forgot this one.

No need to worry about, I will test (if necessary fix) it in a follow-up PR.